generators/gatling/templates/src/test/java/gatling/simulations/_entityClass_GatlingTest.java.ejs

```diff
@@ -41,6 +41,7 @@ import org.slf4j.LoggerFactory;
 /**
  * Performance test for the <%= entityClass %> entity.
  */
+@SuppressWarnings("java:S125")
```

Comment:
I think we should remove the commented code.
I think we can do something like:

```suggestion
 *
 * @see <a href="https://github.com/jhipster/generator-jhipster/tree/v<%- jhipsterVersion %>/generators/gatling#logging-tips">Logging tips</a>
 */
```
Add a README.md in https://github.com/jhipster/generator-jhipster/tree/main/generators/gatling with:
````
## Logging tips

```
LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
{
    // Log all HTTP requests
    //context.getLogger("io.gatling.http").setLevel(Level.valueOf("TRACE"));
    // Log failed HTTP requests
    //context.getLogger("io.gatling.http").setLevel(Level.valueOf("DEBUG"));
}
```
````

Reply:
@mshima ok let's try this